main.py

```
@@ -2050,6 +2050,13 @@ async def sync_upcoming_loop():
     except Exception as _agent_err:
         print(f"⚠️  Agent coordinator failed to start: {_agent_err}")
 
+    # Dispose the engine used for bootstrap to free up connections for the main app
+    try:
+        from app.db.database import engine
+        await engine.dispose()
+        print("♻️  Bootstrap connections recycled")
+    except Exception:
+        pass
```

Ruff (0.15.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)